namelists/echam/6.3.05p2-wiso/NUDG/namelist.echam

```diff
@@ -0,0 +1,131 @@
+&runctl
+    dt_start = 2013, 12, 31, 23, 52, 30
+    dt_stop = 2019, 12, 31, 23, 52, 30
+    putrerun = 1, 'months', 'last', 0
+    lfractional_mask = .false.
+    lresume = .true.
+    out_datapath = './'
+    out_expname = 'NUDGING_ERA5_T63L47'
+    rerun_filetype = 4
+    delta_time = 450
+    putdata = 6, 'hours', 'last', 0
+    lamip = .true.
+    lnudge = .true.
+    lcouple = .false.
+    lcouple_co2 = .false.
+    default_output = .false.
+    l_orbvsop87 = .true.
```

Comment:
We could control the deviations from a standard namelist, needed for nudging through a `choose_with_nudging` switch. In this way we can combine the different scenarios already available with nudging (a more modular approach)

Reply:
Agree with Miguel: we should only get the exact changes required that differ between a standard namelist and a nudging friendly one. I'll make a comparison later this afternoon.